taprpc/rfqrpc/rfq.proto

```diff
@@ -0,0 +1,141 @@
+syntax = "proto3";
+
+package rfqrpc;
+
+option go_package = "github.com/lightninglabs/taproot-assets/taprpc/rfqrpc";
+
+service Rfq {
+    /* tapcli: `rfq buyorder`
+    UpsertAssetBuyOrder is used to upsert a buy order for a specific asset.
+    */
+    rpc UpsertAssetBuyOrder (UpsertAssetBuyOrderRequest)
+        returns (UpsertAssetBuyOrderResponse);
+
+    /* tapcli: `rfq selloffer`
+    UpsertAssetSellOrder is used to upsert a sell offer for a specific asset.
+     */
+    rpc UpsertAssetSellOffer (UpsertAssetSellOfferRequest)
+        returns (UpsertAssetSellOfferResponse);
+
+    /* tapcli: `rfq acceptedquotes`
+    QueryRfqAcceptedQuotes is used to upsert a sell order for a specific
+    asset.
+     */
+    rpc QueryRfqAcceptedQuotes (QueryRfqAcceptedQuotesRequest)
+        returns (QueryRfqAcceptedQuotesResponse);
+
+    /*
+    SubscribeRfqEventNtfns is used to subscribe to RFQ events.
+     */
+    rpc SubscribeRfqEventNtfns (SubscribeRfqEventNtfnsRequest)
+        returns (stream RfqEvent);
+}
+
+message AssetSpecifier {
+    oneof id {
```

Comment:
Just a thought I had while working on something else: Should we just re-use the universe ID here that looks exactly the same? Would semantically more or less make sense as well IMO. Then we could re-use all the parsing/marshaling functions.

Reply:
I think that the universe ID structure should contain the asset specifier structure as a field. The universe ID contains a proof type field which is not relevant when specifying an asset. So I think we should have:

```
message UniverseId {
    AssetSpecifier asset_specifier = 1;

    ProofType proof_type = 2;
}
``` 

Would be good to be able to re-use parsing/marshaling functions :+1: